### DISCUSSION TITLE
Path Loss Models - Showcase

### DISCUSSION THREAD

**Discussion by avarga** [inet-framework/inet#601] [Showcases]
Discuss the "Path Loss Models" showcase here. Ideas and recommendations are welcome.

**Comment by remykif** [comment]
Dear Sir,

In the results file "PathLoss.anf", I did not find the vector of the reception power  "receptionPower" .
Is there any modification to do in order to have the Reception Power of the destination ?

Remy.
Thanks.

**Comment by adamgeorge309** [comment]
The receptionPower statistic is declared in the NED file:
`    @statistic[receptionPower](source="receptionMinSignalPower(radioMedium.signalArrivalStarted)"; record=last);
`
Here, only the last value of the receptionPower is recorded. To record the vectors as well, add `vector`:
`    @statistic[receptionPower](source="receptionMinSignalPower(radioMedium.signalArrivalStarted)"; record=last,vector);
`
But in this showcase, there are lots of runs, and just one packet is sent in each of them. The distance of the nodes is increased incrementally in the runs. So the receptionPower:vector would contain the same data as the receptionPower:last.

**Comment by sihamhasan** [comment]
Hi Avarga

I would like to save the .ini and .ned files from Path Loss Models at the link https://inet.omnetpp.org/docs/showcases/wireless/pathloss/doc/index.html#:~:text=Path%20loss%20models%20are%20used,propagates%20away%20from%20the%20transmitter.&text=The%20default%20path%20loss%20model,%2Dof%2Dsight%20propagation%20path.

Is it possible to import to oment++  to run it?

Thanks in advance for answering my question in advance and stay safe
Siham
